vuepress/docs/next/docs/curate/bundle-versions-faq.md

```diff
@@ -1,77 +1,57 @@
+## How-Tos
+### 1. How do I create a new version of a bundle?
+To release new versions of your bundle after changes have been made, edit the `entando.json` with the new version number for the bundle, then pack and publish your images to Docker. Docker will provide tags to update the bundle version. Once you deploy and install the bundle, the new version number will appear in the App Builder and automatically update in your apps. 
 
-## What format should I use to version my bundle?
+Micro frontends and microservices can have their own version numbers, independent of the bundle version, and can be updated in the same way.
 
-Entando requires versions to follow the [semantic versioning 2.0.0](https://semver.org/#semantic-versioning-200), with the possibility to prepend a `v` to the version itself. Some valid bundle versions are:
+### 2. My bundle contains a microservice generated with the Entando Component Generator; does the version of the microservice have to be the same as the bundle version?
 
-- 1.0
-- v0.1.0-alpha
-- 1.0.1-SNAPSHOT
-- 22.109.10234-RC.1
+The version of the microservice - or the Docker image associated with the microservice - isn't bound to the version of the bundle containing the microservice itself. 
 
+This gives the developer control over the bundle release process, especially in situations where the bundle may contain many components.
 
-## As a bundle developer, how should I create a new version of a bundle?
-If you're a bundle developer and you want to release a new version of your bundle, you can simply update the content of your bundle and commit the changes to the bundle Git repository. 
-Once all the required changes are committed, you can create a new tag in the Git repository using the `git tag` command and publish it to the remote repository using the `git push --tags` command
 
-For example, let's assume my bundle already has a version `1.0.0` and I want to publish version `2.0.0` of my bundle; here are the commands I need to use to create and publish the new tag:
+## Conventions 
+ 
+### 1. How is a bundle version defined?
 
+Bundle versions are defined by the creator and set in the bundle descriptor `entando.json`. You can have multiple versions of a bundle as long as they are defined and tagged as such.
+ 
+```json
+{
+  "name": "my-bundle",
+  "description": "This is a description of my-bundle",
+  "type" : "bundle",
+  "version": "1.0.0",
+  "svc": [...]
+  ...
+
+}
 ```
-    git tag -a "2.0.0" -m "My new version"
-    git push --tags
-```
+### 2. What format should I use to version my bundle?
 
-## How to prevent a particular bundle version from being visible in the Local Hub?
+Follow the recommended [semantic versioning 2.0.0](https://semver.org/#semantic-versioning-200), with the option to prepend a `v` to the number. Some valid bundle versions are:
 
-If, for some reason, you don't want a particular version to be available for the installation, you can proceed as follows:
+- 1.0
```

Comment:
My bad on the earlier pass, we require x.y.z (or vx.y.z) so this line should be 1.0.0